ios/brave-ios/Tests/BraveSharedTests/URLExtensionTests.swift

```diff
@@ -260,8 +260,8 @@ class URLExtensionTests: XCTestCase {
   func testBaseDomain() {
     XCTAssertEqual(URL(string: "http://brave.com")?.baseDomain, "brave.com")
     XCTAssertEqual(URL(string: "https://brave.com")?.baseDomain, "brave.com")
-    XCTAssertEqual(URL(string: "http://community.brave.com")?.baseDomain, "brave.com")
-    XCTAssertEqual(URL(string: "https://community.brave.com")?.baseDomain, "brave.com")
+    XCTAssertEqual(URL(string: "http://community.brave.app")?.baseDomain, "brave.app")
+    XCTAssertEqual(URL(string: "https://community.brave.app")?.baseDomain, "brave.app")
```

Comment:
You can leave these URLs as-is for the unit test, `brave.app` is found in `effective_tld_names.dat` so the `baseDomain` will return `community.brave.app` instead of just `brave.app`